src/Utils/Auth/AuthUtil.ts

```diff
@@ -0,0 +1,21 @@
+const isAuthenticated = () => {
+  // Look for the `auth` cookie
+  const authCookie = document.cookie
+    .split("; ")
+    .find((row) => row.startsWith("auth="));
+
+  // Check that the cookie is present and not empty
+  if (authCookie) {
+    const authValue = authCookie.split("=")[1];
+    return authValue !== undefined && authValue !== "";
+  }
+
+  return false;
+};
+
+const logout = () => {
```

Comment:
The logout logic has been nicely exported here, I suggest we do the same for the login logic.